apps/dotcom/src/app/pages/faq/Accordion.tsx

```diff
@@ -0,0 +1,66 @@
+// Mostly copied from src/app/components/Accordion.tsx
+
+import React, { useState } from 'react';
+import {
+  Accordion as MuiAccordion,
+  AccordionDetails,
+  AccordionSummary,
+  Typography,
+  Fade,
+} from '@material-ui/core';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { makeStyles } from '@material-ui/core/styles';
+
+interface AccordionProps {
+  sections: any[];
```

Comment:
It might be beneficial to create a section type to be used here in place of any[]

Reply:
+1 Possible type might be something like
```tsx
interface Section {
  title: string,
body: React.Element
}
```